monkey/infection_monkey/monkey.py

```diff
@@ -119,6 +118,10 @@ def start(self):
 
             self.shutdown_by_not_alive_config()
 
+            if self.is_started_on_island():
+                ControlClient.report_start_on_island()
+            ControlClient.should_monkey_run(self._opts.vulnerable_port)
```

Comment:
What happens when it's not specified, such as a manual run?

Reply:
`should_monkey_run` returns True. But we can avoid sending the request, which it currently does!